lib/eclipse/Deck/DeckKeyword.cpp

```diff
@@ -17,9 +17,9 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "DeckKeyword.hpp"
```

Comment:
I take it back - the moving should coincide with the makefile change because it's a reorganisation as well. It's touching the files, sure, but only by `git mv`, so it shouldn't be that big of a deal.